wrappers/python/setup.py

```diff
@@ -5,63 +5,86 @@
 import re
 import os
 import sys
+import shutil
 import platform
-from distutils.core import setup
 
-MAJOR_VERSION_NUM='@OPENMM_MAJOR_VERSION@'
-MINOR_VERSION_NUM='@OPENMM_MINOR_VERSION@'
-BUILD_INFO='@OPENMM_BUILD_VERSION@'
+try:
+    from setuptools import setup, Extension
+except ImportError:
+    from distutils.core import setup, Extension
+
+from distutils.command.install_lib import install_lib as _install_lib
+
+MAJOR_VERSION_NUM = '@OPENMM_MAJOR_VERSION@'
+MINOR_VERSION_NUM = '@OPENMM_MINOR_VERSION@'
+BUILD_INFO = '@OPENMM_BUILD_VERSION@'
 IS_RELEASED = False
 
 __author__ = "Peter Eastman"
 __version__ = "%s.%s" % (MAJOR_VERSION_NUM, MINOR_VERSION_NUM)
 
+
 def reportError(message):
     sys.stdout.write("ERROR: ")
     sys.stdout.write(message)
     sys.stdout.write("\nExiting\n")
     sys.exit(1)
 
-def removeRecursive(dir):
-    for file in os.listdir(dir):
-        path = os.path.join(dir, file)
-        if os.path.isdir(path):
-            removeRecursive(path)
+
+def libraryFilename(libName):
+    if platform.system() == 'Windows':
+        return '%s.dll' % libName
+    elif platform.system() == 'Darwin':
+        return 'lib%s.dylib' % libName
+    elif platform.system() == 'Linux':
+        return 'lib%s.so' % libName
+    else:
+        raise ValueError('Unknown platform: "%s"' % platform.system())
+
+
+def getLibraries():
+    libraries = ['OpenMM',
```

Comment:
no, cmake won't let you build the python wrappers if those libraries are turned off.